cypress/e2e/displays.cy.js

```diff
@@ -29,14 +29,18 @@ describe('Displays', function () {
 
         cy.visit('/display/view');
 
-        cy.get('#displays_wrapper.dataTables_wrapper').check();
+        cy.wait(500);
```

Comment:
The use of wait isn't really recommended if there's any requests happening, a slower system could take more than the 500ms and make the test fail. The possible solution for this is to have a custom wait flag for a specific request on top of the test, and make the next step wait for the request to be finished.

### To define the waiting variable:
```
cy.server();
cy.route('**/path/*').as('requestName');
```

### When you need to use wait ( instead of a time in ms ):
```
cy.wait('@requestName');
```

This functionality will need to be replaced in all Cypress tests for `cy.intercept()` though, so if you want to implement that instead: https://docs.cypress.io/guides/references/migration-guide#Migrating-cy-route-to-cy-intercept